frontend/src/components/form/api/ApiCheckbox.vue

```diff
@@ -23,7 +23,7 @@ Displays a field as a v-checkbox + write access via API wrapper
 <script>
 import { apiPropsMixin } from '@/mixins/apiPropsMixin'
 import ApiWrapper from './ApiWrapper'
-import StatusIcon from './StatusIcon'
+import StatusIcon from '../base/StatusIcon'
```

Comment:
Das StatusIcon wird aktuell nur bei ApiInputs angezeigt oder? Sollten wir das auch noch in den form/api Unterordner verschieben?